client/selfrepair/normandy_driver.js

```diff
@@ -128,10 +128,27 @@ export default class NormandyDriver {
     }
   }
 
+  /**
+   * Generate a fresh UUID
+   * @return {String} Generated UUID
+   */
   uuid() {
     return uuid.v4();
   }
 
+  /**
+   * Get a user id. If one doesn't exist yet, make one up and store it in local storage.
+   * @return {String} A stored or generated UUID
+   */
+  get userId() {
```

Comment:
Adding this to the driver should be reflected in [the driver docs](http://normandy.readthedocs.io/en/latest/dev/driver.html).